src/dev/mergince.md

```diff
@@ -27,18 +27,18 @@ $ sudo chmod g+s ./projects/
 $ docker-compose -f docker-compose.yml up
 ```
 ​​
-### Initialise
+### Initialise database
 If server is started for the first time, database needs to be initialised and super-user created (set admin username, password and email):
 ```shell
 $ docker exec merginmaps-server flask init-db
 $ docker exec merginmaps-server flask user create <username> <password> --is-admin --email <email>
 ```
 ​
-### Configure
+### Configure environment
 There are several application settings which can be changed via <GitHubRepo desc="config variables" id="MerginMaps/server/blob/master/.prod.env" />. Some of them have defaults and some of them needs to be modified, particularly those with `fixme` placeholder (marked with asterisks below).
 ​
 #### Server basics
-Settings about your deployment. Variables marked with star ⭐️ need to be modified.
+Variables marked with star ⭐️ need to be modified.
```

Comment:
I think we should clarify what this really means. One thing is required settings without local deployment would not work (maybe we should make sure there is everywhere some default). Another thing is that we should clearly state which values should be modified for production deployment (e.g. default value is not safe, not recommended)

Reply:
Agreed, let me do that